pkg/distributors/contracts/MultiDistributor.sol

```diff
@@ -272,8 +272,8 @@ contract MultiDistributor is IMultiDistributor, ReentrancyGuard, MultiDistributo
 
             // Checked arithmetic is not required due to the if
             uint256 remainingTime = periodFinish - block.timestamp;
-            uint256 leftoverTokens = Math.mul(remainingTime, distribution.paymentRate);
-            distribution.paymentRate = Math.divDown(amount.add(leftoverTokens), duration);
+            uint256 leftoverTokens = FixedPoint.mulDown(remainingTime, distribution.paymentRate);
```

Comment:
We should have a comment here mentioning that fixed point multiplication of fixed (rate) and non-fixed (time) params results in a non-fixed result.